retrieval_service/models/models.py

```diff
@@ -38,9 +38,43 @@ class Amenity(BaseModel):
     terminal: str
     category: str
     hour: str
+    sunday_start_hour: Optional[datetime.time]
```

Comment:
Does this need to be Optional?

Reply:
Yes, if the store is not operating on that day, this value will be None.